__tests__/lint/assertions-ref-pattern-error/components/schemas/Owners.yaml

```diff
@@ -0,0 +1,27 @@
+get:
```

Comment:
I would move those files out of `components/schemas` folder into `components/pathItems`.